include/Model/Languages.php

```diff
@@ -303,21 +303,27 @@ public function update( $args ) {
 			}
 
 			// Update menus locations.
-			if ( ! empty( $this->options['nav_menus'] ) ) {
-				foreach ( $this->options['nav_menus'] as $theme => $locations ) {
+			if ( ! empty( $this->options->get( 'nav_menus' ) ) ) {
+				$nav_menus = $this->options->get( 'nav_menus' );
```

Comment:
```suggestion
                         $nav_menus = $this->options->get( 'nav_menus' );
			if ( ! empty( $nav_menus ) ) {
```
I'm surprised PHPStan doesn't throw an error. How does he know the method is pure?

Reply:
Why would PHPStan throw an error?
I didn't do that because `get()` is cheap, but I don't mind changing.

Reply:
PHPStan should throw an error because to consecutive calls to a user defined function (here `PLL_ABstract_Options::get()` could return different values.
Of course this is not the case here since `PLL_ABstract_Options::get()` is a pure method, but PHPStan doesn't know that.
So I'm surprised it doesn't throw any error.
You could kept the code as it is, it was more a general question.